src/components/views/rooms/RoomDetailList.js

```diff
@@ -49,6 +49,7 @@ const RoomDetailRow = React.createClass({
         dis.dispatch({
             action: 'view_room',
             room_id: this.props.room.roomId,
+            room_alias: this.props.room.canonicalAlias || this.props.room.aliases[0],
```

Comment:
`TypeError: Cannot read property '0' of undefined` if room has no aliases (nor canonicalAlias)?

Reply:
oh good point